src/commands/General/Chat Bot Info/invite.ts

```diff
@@ -0,0 +1,28 @@
+import { SkyraCommand, SkyraCommandOptions } from '@lib/structures/SkyraCommand';
+import { ApplyOptions } from '@skyra/decorators';
+import { KlasaMessage } from 'klasa';
+import { PermissionLevels } from '@lib/types/Enums';
+
+@ApplyOptions<SkyraCommandOptions>({
+	cooldown: 10,
+	description: language => language.tget('COMMAND_INVITE_DESCRIPTION'),
+	extendedHelp: language => language.tget('COMMAND_INVITE_EXTENDED'),
+	usage: '[noperms]',
+	flagSupport: true,
+	guarded: true
+})
+export default class extends SkyraCommand {
+
+	public async run(message: KlasaMessage, [noperms]: ['noperms' | undefined]) {
+		if (noperms === 'noperms' || Reflect.has(message.flagArgs, 'noperms') || Reflect.has(message.flagArgs, 'nopermissions')) {
```

Comment:
Second condition is never true, if a flag with the name of a parameter is given, it'll give its value to it.